python/ray/experimental/client/worker.py

```diff
@@ -59,6 +66,38 @@ def _put(self, val):
         resp = self.server.PutObject(req)
         return ClientObjectRef(resp.id)
 
+    def wait(self,
+             object_refs: List[ClientObjectRef],
+             *,
+             num_returns: int = 1,
+             timeout: float = None
+             ) -> (List[ClientObjectRef], List[ClientObjectRef]):
+        assert isinstance(object_refs, list)
+        for ref in object_refs:
+            assert isinstance(ref, ClientObjectRef)
+        data = {
+            "object_refs": [
+                cloudpickle.dumps(object_ref) for object_ref in object_refs
```

Comment:
Instead of encoding with cloudpickle, can we use the raw bytes here? `object_ref.binary()`

Reply:
Not really, note that the type of object ref is not ObjectRef but rather it is ClientObjectRef.

Reply:
We can do that on the server-side though. I changed the server-side to not dump the returned ids and hence we do not have to load them on the worker side.